configs/components/rubygem-mini_portile.rb

```diff
@@ -0,0 +1,15 @@
+component "rubygem-mini_portile" do |pkg, settings, platform|
+  pkg.version "0.6.2"
+  pkg.md5sum "281cc0d974d3810d1195ad4a863ba5b6"
+  pkg.url "http://buildsources.delivery.puppetlabs.net/mini_portile-#{pkg.get_version}.gem"
+
+  pkg.build_requires "ruby"
+
+  # Because we are cross-compiling on ppc, we can't use the rubygems we just built.
+  # Instead we use the host gem installation and override GEM_HOME. Yay?
+  pkg.environment "GEM_HOME" => settings[:gem_home]
```

Comment:
This is set for all gems everywhere, so it hasn't broken anything yet. As long as we set https://github.com/puppetlabs/puppet-agent/blob/master/configs/projects/puppet-agent.rb#L25 correctly, it doesn't look like it does anything bad